src/main/java/controllers/kvis/dao/KvisActivationDAO.java

```diff
@@ -3,16 +3,46 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import controllers.KvisDatabase;
 import controllers.Table;
+import controllers.kvis.dto.kvisAPI.KvisActivatedAPIDTO;
 
 import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.util.ArrayList;
 
 /**
  * @author Alfred Röttger Rydahl
  * @date 22/11/2021
  **/
 public class KvisActivationDAO
 {
+	public static KvisActivatedAPIDTO[] getAllActiveKvisses() throws SQLException, JsonProcessingException
+	{
+		// Query String
+		final String query = String.format("SELECT * FROM %s", Table.ACTIVE_KVIS.TableName);
+		
+		// Execute and parse
+		return KvisDatabase.queryDatabase(
+				query,
+				new IResultSetParser<KvisActivatedAPIDTO[]>()
+				{
+					@Override
+					public KvisActivatedAPIDTO[] parse(ResultSet resultSet) throws SQLException, JsonProcessingException
+					{
+						ArrayList<KvisActivatedAPIDTO> list = new ArrayList<>(2);
+						while(resultSet.next())
+						{
+							list.add(new KvisActivatedAPIDTO(
+									resultSet.getString("kvis_id"),
+									resultSet.getString("interaction_code"),
+									resultSet.getTimestamp("activated_time")
+							));
+						}
+						return list.toArray(new KvisActivatedAPIDTO[0]);
```

Comment:
Ville det ikke være nemmere bare at returnere en liste så?

Reply:
Ja, det er rigtigt. Grunden til at jeg har gjort det sådan her er, at jeg ville være sikker på at der ikke kommer andet med i det response. Det kan godt være at det ikke gør nogen forskel med en liste, men jeg ved det faktisk ikke :smile:

Reply:
I .Net APIer laver man i hvert fald lister, men hvis det ikke irritere dig, så er det fint for mig 👍